public/locales/en/translation.json

```diff
@@ -32,6 +32,68 @@
     "NAME": "NAME",
     "DATE OF BIRTH": "DATE OF BIRTH",
     "VACCINATION RECORD": "VACCINATION RECORD"
+  },
+  "footer": {
+    "FAQ": "FAQ",
+    "Privacy": "Privacy"
+  },
+  "faq": {
+    "Frequently asked questions": "Frequently asked questions",
+    "items": [
+      {
+        "question": "What is a SMART\u00AE Health Card and where can I get one?",
+        "answer": "SMART\u00AE Health Cards are paper or digital versions of your clinical information, such as vaccination history or test results. They allow you to keep a copy of your records on hand and easily share this information with others if you choose. Please visit <a href=\"https://smarthealth.cards/en/\">https://smarthealth.cards</a> for more information and details on how to obtain a SMART\u00AE Health Card."
+      },
+      {
+        "question": "What does it mean if you have an unsupported QR code?",
+        "answer": "An unsupported QR code can mean that the QR code you are attempting to scan is unreadable, is not a SMART\u00AE Health Card compliant QR code, or is a SMART\u00AE Health Card that does not contain vaccination data. The organization that provided your SMART\u00AE Health Card may be able to help you obtain a new QR code."
+      },
+      {
+        "question": "What does it mean if your SMART\u00AE Health Card is 'Verified'?",
+        "answer": "This means that the QR code or codes you scanned are valid SMART\u00AE Health Cards from a recognized issuing organization."
```

Comment:
Should we add an FAQ question or description somewhere explaining what a "recognized issuing organization" is, since it comes up in 3 of the current FAQ questions?

I agree with @sarahmcdougall's comment that we should meet as a group to discuss what specific questions we eventually want to add to the FAQ page. This might be especially helpful after we discuss the different error handling scenarios, so I would be fine pushing this off until then.